backend/src/database/migrations/20220324200600_create_contacts_table.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
```

Comment:
There isn't a practical difference between arrow functions and the old syntax in this implementation but we should maybe have a team discussion about JS syntax in our apps going forward. @ShawnTurple @CR-gbc

Reply:
the migration generator tool makes this code for you. @ASpiteri-BCGov  is making a research ticket to explore how we can make that generator make our desired syntax. because this is only for the technically separate seeder system, i am ok with leaving the classic syntax in place.

Reply:
Thanks for the context! Glad modern syntax is being worked on